camera.cpp

```diff
@@ -178,4 +199,26 @@ void Camera::SetupViewMatrix(void){
     view_matrix_ *= trans;
 }
 
+void Camera::SwitchCameraMode(void) {
+
+	if (camera_mode_ == CameraMode::FIRST_PERSON) {
+		camera_mode_ = CameraMode::THIRD_PERSON;
+
+	}
+	else {
+		camera_mode_ = CameraMode::FIRST_PERSON;
+	}
+
+	RepositionTetheredChild(GetChild("Ship"));
+}
+
+void Camera::RepositionTetheredChild(SceneNode * child) {
+	if (camera_mode_ == CameraMode::THIRD_PERSON) {
```

Comment:
RepositionTetheredChild I don't believe is being used anymore, based on how we fixed the camera.